addons/hr_expense/tests/test_expenses.py

```diff
@@ -1148,3 +1095,36 @@ def test_expense_from_attachments(self):
         self.env['hr.expense'].create_expense_from_attachments(attachment.id)
         expense = self.env['hr.expense'].search([], order='id desc', limit=1)
         self.assertEqual(expense.account_id, product.property_account_expense_id, "The expense account should be the default one of the product")
+
+    def test_expense_payment_method(self):
+        new_payment_method_line = default_payment_method_line = self.company_data['default_journal_bank'].outbound_payment_method_line_ids[0]
+        check_method = self.env['account.payment.method'].search([('code', '=', 'check_printing'), ('payment_type', '=', 'outbound')], limit=1)
+        if check_method:
```

Comment:
Why is this dependent on the data of the database?
Test should be independent.

Reply:
I improved it, I was trying to keep some of the old test idea but it was not a good idea indeed.
I need to test when you change the payment method so I'll create a new one instead.

Reply:
test is red now